robolectric-resources/src/main/java/org/robolectric/manifest/AndroidManifest.java

```diff
@@ -152,6 +152,18 @@ void parseAndroidManifest() {
         }
 
         minSdkVersion = getTagAttributeIntValue(manifestDocument, "uses-sdk", "android:minSdkVersion");
+
+        String targetSdkText = getTagAttributeText(manifestDocument, "uses-sdk",
+            "android:targetSdkVersion");
+        if (targetSdkText != null) {
+          // Support Android O Preview. This can be removed once Android O is officially launched.
+          if (targetSdkText.length() == 1 && targetSdkText.equals("O")) {
+            targetSdkVersion = 26;
```

Comment:
Why do we  need to find the value of `targetSdkVersion` if we later re-assign it be another value?